articles/azure-functions/deployment-zip-push.md

```diff
@@ -25,7 +25,7 @@ To speed up development, you might find it easier to deploy your function app pr
 For more information, see the [.zip deployment reference](https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file).
 
 >[!IMPORTANT]
-> When you use .zip deployment, any files from an existing deployment that aren't found in the .zip file are deleted from your function app.  
+> When you use .zip deployment, any files from an existing deployment that aren't found in the .zip file are deleted from your function app.Any other files and directories found in the site that aren't being overwritten by the deployment, such as those placed there via FTP or created by your app during runtime, will be preserved.
```

Comment:
This was a bit confusing as the two sentences seem contradictory at first glance. 
If I understand correctly, the first sentence states that any files not found in the zip will be deleted. The second sentence states that certain files and directories uploaded by FTP or created by the app with be preserved nonetheless. If so, let's focus the second sentence on what causes certain files/directories to be preserved.

i.e.,
"...app. However, any files and directories added to the site via FTP or by your app during runtime will be preserved."

^ Is this technically correct?

Reply:
Yes, this is correct based on the below link : 

https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file-or-url#comparison-with-zip-api


When a new build is deployed with zipdeploy, files and directories that were created by the previous deployment(made via zipDeploy) but are no longer present in the build will be deleted. **Any other files and directories found in the site that aren't being overwritten by the deployment, such as those placed there via FTP or created by your app during runtime, will be preserved. e.g. a previous folder 'BackupLogs' created from an FTP upload or manual creation would be preserved even if the zip doesn't contain this folder.**

Reply:
Can we use this? 

When you use .zip deployment, any files and directories from the previous deployment that aren't found in the deployed .zip file will be deleted from your function app. Files and directories that do not originate from a .zip deployment, such as those added to the site via FTP or by your app during runtime, will be preserved.

Reply:
^ just making clear what files/directories get deleted and what gets preserved.
I don't think we need to the example for brevity sake.